example/lib/main.dart

```diff
@@ -268,8 +268,25 @@ class _HomePageState extends State<HomePage> with WidgetsBindingObserver {
     Pspdfkit.setLicenseKey("LICENSE_KEY_GOES_HERE");
   }
 
+  void flutterPdfActivityOnPauseHandler() {
+    print("flutterPdfActivityOnPauseHandler");
+  }
+
+  void pdfViewControllerWillDismissHandler() {
+    print("pdfViewControllerWillDismissHandler");
+  }
+
+  void pdfViewControllerDidDismissHandler() {
+    print("pdfViewControllerDidDismissHandler");
+  }
+
   @override
   Widget build(BuildContext context) {
+    Pspdfkit.setupPlatformCallHandler();
```

Comment:
Is there a reason for outsourcing the setup call to the user of our library? If so, we have to add documentation about this. Otherwise, let's internalize this call, and make it part of the library's initialization routines.

Reply:
> Is there a reason for outsourcing the setup call to the user of our library?

Nope. I didn't know better. 

> Otherwise, let's internalize this call, and make it part of the library's initialization routines.

Done!